content/millennium/r4/general-clinical/condition.md

```diff
@@ -0,0 +1,79 @@
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id)
+* [Clinical status](http://hl7.org/fhir/R4/condition-definitions.html#Condition.clinicalStatus)
+* [Verification status](http://hl7.org/fhir/R4/condition-definitions.html#Condition.verificationStatus)
+* [Category](http://hl7.org/fhir/R4/condition-definitions.html#Condition.category)
+* [Severity](http://hl7.org/fhir/R4/condition-definitions.html#Condition.severity)
+* [Condition code](http://hl7.org/fhir/R4/condition-definitions.html#Condition.code)
+* [Patient](http://hl7.org/fhir/R4/condition-definitions.html#Condition.subject)
+* [Patient encounter when first recorded (only applies to diagnoses)](http://hl7.org/fhir/R4/condition-definitions.html#Condition.encounter)
+* [Onset (dateTime)](http://hl7.org/fhir/R4/condition-definitions.html#Condition.onset_x_)
+* [Resolved date  (only applies to problems)](http://hl7.org/fhir/R4/condition-definitions.html#Condition.abatement_x_)
+* [Date recorded](http://hl7.org/fhir/R4/condition-definitions.html#Condition.recordedDate)
+* [Who recorded the condition](http://hl7.org/fhir/R4/condition-definitions.html#Condition.recorder)
+* [Comment/Note](http://hl7.org/fhir/R4/condition-definitions.html#Condition.note)
+
+## Terminology Bindings
+
+<%= terminology_table(:condition, :r4) %>
+
+## Search
+
+Search for Conditions that meet supplied query parameters:
+
+    GET /Condition?:parameters
+
+_Implementation Notes_
+
+* Currently `problem-list-item` is supported.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name               | Required?          | Type          | Description
+--------------------|--------------------|---------------|-----------------------------------------------------------------------
+ `patient`          | This or `subject`  | [`reference`] | Who the sensitivity is for. Example: `12345`
+ `subject`          | This or `patient`  | [`reference`] | Who the sensitivity is for. Example: `12345`
+ `clinical-status`  | No                 | [`token`]     | The clinical status of the condition. Example: `active`, `inactive`, `resolved`
+ `category`         | No                 | [`token`]     | The category of the condition. Category problem-list-item is supported as of now. Example: `problem-list-item`, `encounter-diagnosis`
+
+### Headers
+
+ <%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Condition?patient=1316024
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_condition_bundle) %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+[`reference`]: https://hl7.org/fhir/r4/search.html#reference
+[`token`]: http://hl7.org/fhir/R4/search.html#token
```

Comment:
should this also be a https:// url?

Reply:
Fixed. Throughout. [4872fd1]

Reply:
the token link is still http instead of https.

Reply:
oh. missed that. Fixed. [178fb7f]